docs/plugins/analysis-nori.asciidoc

```diff
@@ -434,3 +439,105 @@ Which responds with:
 --------------------------------------------------
 
 <1> The Hanja form is replaced by the Hangul translation.
+
+
+[[analysis-nori-number]]
+==== `nori_number` token filter
+
+The `nori_number` token filter normalizes Korean numbers
+to regular Arabic decimal numbers in half-width characters.
+
+Korean numbers are often written using a combination of Hangul and Arabic numbers with various kinds punctuation.
+For example, ３．２천 means 3200.
+This filter does this kind of normalization and allows a search for 3200 to match ３．２천 in text,
+but can also be used to make range facets based on the normalized numbers and so on.
+
+Notice that this analyzer uses a token composition scheme and relies on punctuation tokens
```

Comment:
Maybe add add a `NOTE:` to emphasize this part ?

Reply:
Done!